FlickrViewer/FlickrViewer/BDBO/BDBOUser.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  BDBOUser.swift
+//  FlickrViewer
+//
+//  Created by Kirill Shteffen on 25/07/2018.
+//  Copyright © 2018 BlackBricks. All rights reserved.
+//
+
+import UIKit
+
+class BDBOUser{
```

Comment:
BDBO is prefix for BDBOAuth1Manager library classes. You can use just User instead